src/context.rs

```diff
@@ -101,7 +128,41 @@ impl Context {
 
     //fn ffi::cairo_get_target (cr: *mut cairo_t) -> *mut cairo_surface_t;
 
-    pub fn push_group(&self){
+    /// Temporarily redirects drawing to an intermediate surface known as a group. The
+    /// redirection lasts until the group is completed by a call to Context::pop_group()
+    /// or Context::pop_group_to_source(). These calls provide the result of any drawing
+    /// to the group as a pattern, (either as an explicit object, or set as the source
+    /// pattern).
+    /// 
+    /// This group functionality can be convenient for performing intermediate compositing.
+    /// One common use of a group is to render objects as opaque within the group, (so that
+    /// they occlude each other), and then blend the result with translucence onto the
+    /// destination.
+    /// 
+    /// Groups can be nested arbitrarily deep by making balanced calls to Context::push_group()
+    /// / Context::pop_group(). Each call pushes  /pops the new target group onto/from a stack.
+    /// 
+    /// The Context::push_group() function calls Context::save() so that any changes to the
+    /// graphics state will not be visible outside the group, (the pop_group functions call
+    /// Context::restore()).
+    /// 
+    /// By default the intermediate group will have a content type of Content::ColorAlpha.
+    /// Other content types can be chosen for the group by using
+    /// Context::push_group_with_content() instead.
+    /// 
+    /// As an example, here is how one might fill and stroke a path with translucence, but
+    /// without any portion of the fill being visible under the stroke:
+    /// 
+    /// ```
```

Comment:
We might want this to be "ignore" so `cargo test` won't barf on it. (https://doc.rust-lang.org/book/documentation.html#running-documentation-tests)